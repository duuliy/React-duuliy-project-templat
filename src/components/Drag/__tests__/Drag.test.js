
import React, { Fragment } from 'react';
import {shallow,mount,render} from 'enzyme';
import  Drag  from '../Drag.js';
import renderer from 'react-test-renderer';

describe('<Drag/>',  ()=> {
  const corpLangDirecInfo={
    title:"测试",
    visible:true
  }
  it('shoulder render a Drag', function () {
    let app = renderer.create(
        <Fragment>
          <Drag gba={corpLangDirecInfo}/>
        </Fragment>
        ).toJSON();
    expect(app).toMatchSnapshot();
  });
})


