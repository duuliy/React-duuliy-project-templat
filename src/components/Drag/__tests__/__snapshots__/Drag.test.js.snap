// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Drag/> shoulder render a Drag 1`] = `
<p
  className="Drag"
  onMouseDown={[Function]}
/>
`;
